client/blocks/importer/wordpress/import-everything/migration-progress/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { ProgressBar } from '@automattic/components';
+import { MigrationStatus } from '@automattic/data-stores';
+import { Progress, SubTitle, Title } from '@automattic/onboarding';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback } from 'react';
+import { EVERY_FIVE_SECONDS, Interval } from 'calypso/lib/interval';
+import useBackupTitle from './use-backup-title';
+import useRestoreTitle from './use-restore-title';
+import type { MigrationState } from 'calypso/blocks/importer/wordpress/types';
+
+interface Props {
+	status: MigrationStatus;
+	fetchStatus: () => void;
+	details: MigrationState;
+}
+export const MigrationProgress = ( props: Props ) => {
+	const translate = useTranslate();
+	const { details, status, fetchStatus } = props;
+	const backupTitle = useBackupTitle( details );
+	const restoreTitle = useRestoreTitle( details );
+
+	const getTitle = useCallback( () => {
+		switch ( status ) {
+			case MigrationStatus.NEW:
+			case MigrationStatus.BACKING_UP_QUEUED:
+				return translate( 'Backing up your data…' );
+
+			case MigrationStatus.BACKING_UP:
+				return backupTitle;
+
+			case MigrationStatus.RESTORING:
+				return restoreTitle;
+
+			default:
+				return '';
+		}
+	}, [ status, backupTitle, restoreTitle ] );
+
+	return (
+		<Progress className="onboarding-progress-simple">
+			<Interval onTick={ fetchStatus } period={ EVERY_FIVE_SECONDS } />
```

Comment:
When I took over this PR, this had already been changed by @zaerl.

I agree that it's much better to have 5 second intervals since a lot of steps can be changed in that period, especially Restoring process steps. Since we are trying to show more granular progress, this makes sense to me.

Reply:
I lowered it because otherwise, during the first tests, I missed 50% of the steps from BE.

Five is ok, but it can be moved back to 10 like other loops.

Reply:
I would leave it as it is at 5 seconds in order to show as much different steps as we can.

Reply:
Yeah, since we now have more steps it's nice to show more detailed information, thanks for the explanation!